### ISSUE TITLE
A11y_AzureDataStudio(Schema,Dacpac,Backup)_RestoreDatabaseDialog_RestoreDatabaseGeneralTab_Keyboard: Keyboard Focus shifts to top of screen after expanding the combo box

### ISSUE THREAD

**Issue by AbhiramDeshmukh** [microsoft/azuredatastudio#6750]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHub Tags: #A11yMAS; #A11yTCS; #A11y_ADS_Schema_Dacpac_Backup; #Azure Data Studio; #DesktopApp; #Win10; #Keyboard; #NVDA; #MAS1.3.2; #MAS2.4.3;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.10.0-insider (user setup)
Commit: 1ff9a2e
Date: 2019-08-09T07:49:00.254Z
VS Code 1.37.0
Electron: 4.2.7
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

**Additional Details:**
Tool Name: Keyboard, NVDA
MAS References: MAS 2.4.3, MAS 1.3.2

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Connections from left menu.
4. Expand the server connected in step 2.
5. Right click an on prem database and select Restore from Context menu.
6. Start navigating on page using keyboard tab key.
7. Navigate till Database or Restore from Combo box & hit enter or space bar to expand it.
8. Press tab key again & check focus order.

**Actual:**
After expanding combo box when we again hit tab key, keyboard focus shifts to General, Files & Options tab.
Same issue is also observed for combo box available on below screens:
1. Restore Database Options tab
2. Backup Database Dialog

**Expected:**
After expanding combo box when tab key hit, keyboard focus should shift to next control and not to top of screen. Focus order should be left to right & top to bottom. 

**User Impact:**
Keyboard only users have to navigate again from top to combo box. Also screen reader users would not get correct idea about the content present on the screen as screen reader focus also shifts to top.

**Recommendation:**
Please refer to below URL which is repository for accessibility bug fixes.
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference:**
MAS 2.4.3: https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}
MAS 1.3.2: https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={1ee227a7-e3c5-47b8-ab18-48ca8595077b}

**Attachment for reference:**
[6750_A11y_AzureDataStudio(Schema,Dacpac,Backup)_RestoreDatabaseDialog_RestoreDatabaseGeneralTab_Keyboard.pptx](https://github.com/microsoft/azuredatastudio/files/3501786/6750_A11y_AzureDataStudio.Schema.Dacpac.Backup._RestoreDatabaseDialog_RestoreDatabaseGeneralTab_Keyboard.pptx)

**Comment by udeeshagautam** [comment]
Affects all combo-boxes...

**Comment by Benjin** [comment]
Repros with dropdowns in VS Code itself, so marking as upstream.  Filed as https://github.com/microsoft/vscode/issues/79339

**Comment by hannahqin** [comment]
Reopening and reassigning to @fsteffi as per the [Accessibility External Bug Process](https://microsoft.sharepoint.com/teams/ceacc/SitePages/External-Bug-process.aspx). @fsteffi, this is an upstream issue from VS Code. We have already filed this bug with vscode as microsoft/vscode#79339. Per the instructions in the External Bug Process, can you track it there?

**Comment by hannahqin** [comment]
Marking this for Exclusion. @AbhiramDeshmukh, please work with a SME to review.
https://microsoft.sharepoint.com/teams/ceacc/SitePages/External-Bug-process.aspx

**Importance of Experience:** somewhat important
This affects all combo boxes across Azure Data Studio and VS Code (where the bug originated). Combo boxes are most prominently used in wizards that must be installed as part of extensions. 

**Quality of Experience:** annoying
User can tab from top to intended combo box or other element. Alternatively, instead of tabbing from combo box after opening it, user can first hit Enter to close it, then tab out as expected. Workaround is not complicated, but is certainly not ideal and may be annoying for affected users.

**AAT Conformance Doc:** https://cela-ra-aat-prod.azurewebsites.net/#/workflow/ee5c9007-47f4-4fcb-a4d7-a998a617edde?tab=default
**External Remark:** After expanding a combo box using the keyboard and then pressing Tab to move to the next element, keyboard focus will instead shift back to the first element in the User Interface. Keyboard users must manually TAB forward to the intended next element in the User Interface.

**Comment by AbhiramDeshmukh** [comment]
Closing this issue as per the below comment from SME.
"approve this one to close based on the workaround of closing the combobox before tabbing."
